### DISCUSSION TITLE
Add new pages in addition to standard ones in gatsby-starter-portfolio-cara

### DISCUSSION THREAD

**Discussion by jaypinho** [LekoArts/gatsby-themes#1226] [Questions] [answered]
This is likely a very basic question, but how do I add a page in addition to the intro/projects/about/contact ones? I couldn't figure out how to create a new one.

I'd also like to be able to switch around the order of the pages.

Thank you!

**Comment by LekoArts** [comment, marked as answer]
Hi!

Not a basic question so thanks for asking. The theme is most often just used by editing the files inside https://github.com/LekoArts/gatsby-starter-portfolio-cara/tree/master/src/%40lekoarts/gatsby-theme-cara/sections

Anyways, I created a demo repo for you: https://github.com/lekoarts-demos/cara-additional-pages
This should show what you need to know: https://github.com/lekoarts-demos/cara-additional-pages/commit/dbfd6abc4e1c76ab3e868754ae688d52b220c23a

Basically you want to shadow the template that creates the homepage, add a new section and then add it to the homepage.

**Reply by jaypinho** [reply]
Thanks!

I'm curious why the `components` and `sections` folders here are on the same level as `@lekoarts` rather than being on the same level as `@lekoarts/gatsby-theme-cara/templates` (as they are in https://github.com/LekoArts/gatsby-themes/blob/main/themes/gatsby-theme-cara)?

**Reply by LekoArts** [reply]
By creating files inside `@lekoarts/gatsby-theme-cara` you overwrite files from the original theme. The stuff I put into `components` and `sections` is net-new, so it can live in their own folders. This way you don't confuse new content with overwriting existing stuff.

**Reply by jaypinho** [reply]
Thank you!

**Comment by jaypinho** [comment]
Related to reordering, whenever I move around the Projects section, I notice the sky-blue background doesn't move with it. I think I've narrowed it down to this line where the offset is hardcoded in (https://github.com/LekoArts/gatsby-themes/blob/main/themes/gatsby-theme-cara/src/components/projects.tsx#L16). Shouldn't this be a relative offset depending on what offset is passed into the component?

**Reply by LekoArts** [reply]
Yeah, this is a typo, it should use the `offset` variable like here: https://github.com/LekoArts/gatsby-themes/blob/main/themes/gatsby-theme-cara/src/components/projects.tsx#L19

**Comment by nickburczyk** [comment]
Hello! I am new to Gatsby, but I don't see the components folder at all in the downloaded project files. I can cmd+click into the component files, but they don't seem to appear in the project file structure.
<img width="417" alt="image" src="https://github.com/LekoArts/gatsby-themes/assets/33436104/d1a4caf7-c3b8-40b4-a9c1-90bb23f338f4">

**Reply by nickburczyk** [reply]
LOVE the design of this template by the way and am excited to use it!

**Reply by nickburczyk** [reply]
@LekoArts
